eng/Packages.Data.props

```diff
@@ -337,11 +337,30 @@
     <PackageReference Update="xunit.runner.visualstudio" Version="2.4.2" />
     <PackageReference Update="xunit" Version="2.4.2" />
   </ItemGroup>
-
   <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
     <PackageDownload Update="NETStandard.Library.Ref" Version="[2.1.0]" />
   </ItemGroup>
 
+  <!-- IsWcfLibrary is set in "..\sdk\extensions\wcf\Directory.Build.props". This property indicates that WCF/CoreWCF extensions project is being built.-->
+  <ItemGroup Condition="'$(IsWcfLibrary)' == 'true'">
+    <PackageReference Update="CoreWCF.Queue" Version="1.5.2" />
+    <PackageReference Update="System.IO.Pipelines" Version="6.0.3" />
+    <PackageReference Update="Microsoft.AspNetCore" Version="2.1.0" />
+    <PackageReference Update="System.ServiceModel.Primitives" Version="6.2.0" />
+    <PackageReference Update="Azure.Storage.Files.Shares" Version="12.17.1" />
+    <PackageReference Update="Azure.Storage.Queues" Version="12.17.1" />
+  </ItemGroup>
+  <ItemGroup Condition="('$(IsWcfLibrary)' == 'true') AND ($(TargetFramework.StartsWith('net4')))">
+    <PackageReference Update="Microsoft.Extensions.Configuration.UserSecrets" Version="2.1.1" />
+    <PackageReference Update="Microsoft.Extensions.Http" Version="2.1.1" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Configuration" Version="2.1.1" />
+  </ItemGroup>
+  <ItemGroup Condition="('$(IsWcfLibrary)' == 'true') AND (!$(TargetFramework.StartsWith('net4')))">
```

Comment:
You can collapse this one into the main block on 345 and then just use the conditional on 358 to reset the versions for the `net4` special case.